test/tests/optimizationreporter/objective_gradient_minimize/bimaterial/gold/main_out_forward0_data_pt_0001.csv

```diff
@@ -0,0 +1,154 @@
+id,temperature,x,y,z
```

Comment:
we should change the gold file we diff to the output from the optimization reporter.  I think the id's and coordinates might not always be consistent on different computers.  Maybe that is why it is currently diff'ing in the tests.